test/check-application

```diff
@@ -290,17 +291,20 @@ class TestApplication(MachineCase):
     def _filter(self, inp, occ_dict):
         import time
 
+        # allow temporary timestamp failures
+        self.allow_journal_messages(".*timestamp.*")
```

Comment:
What causes this failure? Is this needed?

Reply:
As the input is being entered, it tries to search half-completed timestamps, which results in fail journal messages

Reply:
Okay thanks for the explanation.